### DISCUSSION TITLE
Your browser does not support clipboard

### DISCUSSION THREAD

**Discussion by brcarls** [mealie-recipes/mealie#3430] [General]
When I click to clipboard icon, I get the message "Your browser does not support clipboard"...  I tried Firefox, Chrome and Edge, all the same error....  I guess I need to adjust browser security for this work, any guidance on what exactly mealie needs for this feature?

I'm trying to send a recipe to someone who isn't on my LAN (I don't want to expose my server to the internet).   I can select the Instructions to manually copy and paste, but the Ingredients are not selectable....   I also tried printing to pdf, but there is so much white space that it sprawls to a second page which isn't ideal.   Is there a better way to share via email without putting the server online?

**Comment by Kuchenpirat** [comment]
Copy to clipboard is not supported via insecure connections (http instead of https)

**Comment by iamkirkbater** [comment]
I know this is a _very old_ and resolved issue - but I just recently built a tool that scrapes your Mealie API and outputs them into markdown files. That should make it much easier to copy/paste to send to someone.

The full workflow that I have takes all of these markdown files and uses them to populate a [Hugo](https://gohugo.io/) website so everything gets published online as static HTML pages, so my server isn't exposed.

https://github.com/iamkirkbater/mealie-markdown-exporter

In case anyone comes here with a similar problem domain of trying to keep your sever not-on-the-public-internet.
